src/components/displaySvg/finalSvgDisplay.scss

```diff
@@ -0,0 +1,69 @@
+.display-page-div{
+    padding: 171px;
+    width: 373px; 
+}
+
+  a:hover {
+   background-color: yellow;
+   fill:yellow;
+ }
+
+ svg{
+    max-height: 106px;
+    max-width: fit-content;
+    margin-top: 94px;
+    background-color: white;
+ }
+  
+ .display-svgs-border-match {
+    max-height: 381px;
+    border: outset;
+    border-color: green;
+}
+
+ .display-original-optimised{
+    overflow-y: scroll;
+    max-height: 571px;
+    border: outset;
+    border-color: green;
+ }
+
+ .button-background{
+    background-color: grey;
+    height: 171px;   
+ }
+
+ .loader-text{
+      margin-top: 210px;
```

Comment:
rather use padding

Reply:
I have replaced all margins in the file with padding.